### DISCUSSION TITLE
Pressure Advance - enable in slicer, Klipper, both?

### DISCUSSION THREAD

**Discussion by Stooovie** [OrcaSlicer/OrcaSlicer#2006] [General]
Decoumentation is not clear on this. Where do I enable my tuned PA? In slicer, Klipper or both?

**Comment by LJKraus** [comment]
In OrcaSLicer, edit your filament profile, with Advanced settings enabled:

![settings](https://github.com/SoftFever/OrcaSlicer/assets/139707784/8f5a04da-ce30-4cd5-8bd6-1bd6b203cea0)

**Comment by Stooovie** [comment]
Yes but if I have it disabled in slicer and enabled in Klipper, what happens then? Firmware retraction is a similar kind of confusion but that's two different gcode commands. Thank you!
